app/views/orders/index.html.erb

```diff
@@ -4,4 +4,5 @@
   <%= order.updated_at %>
   <%= order.quantity_of_items %>
   <%= order.grand_total %>
+  <%= order.status %>
 <% end %>
```

Comment:
Can't tell if the user story needs this, but I put it here to pass a test that we may need to change.

Reply:
Re:  status